app/views/documents/_history_notice.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="history-status-block">
+  <h2>This <%= type.downcase %> is from a previous government</h2>
+  <% if government_name %>
```

Comment:
As above, the inside of this partial shouldn't have to know about the internals of a method called around its invocation.  This approach makes the partial self-contained and more safely reusable.